profiler/test/Datadog.Profiler.Native.Tests/LibddprofExporterTest.cpp

```diff
@@ -465,7 +471,51 @@ TEST(LibddprofExporterTest, MustCreateAgentLessExporterIfAgentless)
 
     MetricsRegistry metricsRegistry;
     IAllocationsRecorder* allocRecorder = nullptr;
-    auto exporter = LibddprofExporter(std::move(sampleTypeDefinitions), &mockConfiguration, &applicationStore, runtimeInfo, &enabledProfilers, metricsRegistry, allocRecorder);
+    auto exporter = LibddprofExporter(std::move(sampleTypeDefinitions), &mockConfiguration, &applicationStore, runtimeInfo, &enabledProfilers,
+                                      metricsRegistry, allocRecorder);
+}
+
+TEST(LibddprofExporterTest, MustCollectSamplesFromSamplesProvider)
```

Comment:
Maybe rename to MustCollectSamplesFromProcessProvider

Reply:
will do

Reply:
✔️